### PR TITLE
Create Button component - release/v1.0

### PR DESCRIPTION
**Pull request by JRKKRUPICKI**

### CODE REVIEW

File: billennium/libs/ui/src/lib/button/button.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    title: 'Button'
+} as Meta;
+
+const Template: Story<ButtonProps> = (args) => <Button {...args} />;
+
+export const Create = Template.bind({});
+Create.args = {
+    text: 'Create',
+    backgroundColor: '#FF5A00',
+    theme: 'filled',
```
Comment (polubis) [inline billennium/libs/ui/src/lib/button/button.stories.tsx:15]: theme: motive

File: billennium/libs/ui/src/lib/button/button.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    component: Button,
+    title: 'Button'
+} as Meta;
+
+const Template: Story<ButtonProps> = (args) => <Button {...args} />;
+
+export const Create = Template.bind({});
+Create.args = {
+    text: 'Create',
+    backgroundColor: '#FF5A00',
```
Comment (polubis) [inline billennium/libs/ui/src/lib/button/button.stories.tsx:14]: This should be removed - only motive applies colors and backgrounds

File: billennium/libs/ui/src/lib/button/button.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Reserve.args = {
+    text: 'Reserve',
+    backgroundColor: '#323232',
+    theme: 'filled',
+    shape: 'normal'
+};
+
+export const Add = Template.bind({});
+Add.args = {
+    text: 'Add',
```
Comment (polubis) [inline billennium/libs/ui/src/lib/button/button.stories.tsx:45]: text should be children -> ReactNode

File: billennium/libs/ui/src/lib/button/button.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        padding: 8px 14px;
+    `}
+    ${(props) => props.shape === 'rounded' && css`
+        width: 46px;
+        height: 46px;
+        border-radius: 50%;
+    `}
+`;
+
+const getIcon = (icon: ButtonProps['icon'], textColor: ButtonProps['textColor']) => {
```
Comment (polubis) [inline billennium/libs/ui/src/lib/button/button.tsx:46]: Make it 100% generic like in material UI BUtton -> you can add same API format

**Review by polubis — CHANGES_REQUESTED** [review]
Check comments

**Review by ajronn — CHANGES_REQUESTED** [review]

**Pull request merged**